src/EventListener/InitializeSystemListener.php

```diff
@@ -181,10 +190,7 @@ protected function boot(Request $request = null)
         $this->initializeLegacySessionAccess();
         $this->setDefaultLanguage();
 
-        // Fully load the configuration
-        $objConfig = Config::getInstance();
```

Comment:
No, the whole point about an adapter is that you can use its methods without relying on calling another method first in your tests.